code/src/Artist.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Artist = (props) => {
+  return (
+    <div className="artist">
+      <a href={props.url} className="artistLink">{props.name}</a>
+    </div>
```

Comment:
take away the div around the a here. That will make the artists go on the same row. right now the divs (one for each artist) are rendered inside of a p.